TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj

```diff
@@ -22,8 +22,20 @@
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
     <MtouchArch>i386</MtouchArch>
-    <MtouchLink>None</MtouchLink>
-    <MtouchDebug>true</MtouchDebug>
+    <MtouchLink>SdkOnly</MtouchLink>
```

Comment:
Same issue, I was unable to deploy to the iOS simulator and run/test it out without enabling 'Sdk only' linking.